modules/op-release-notes-1-6.adoc

```diff
@@ -274,4 +273,30 @@ Error from server (InternalError): Internal error occurred: failed calling webho
 // https://issues.redhat.com/browse/SRVKP-1783
 
 * The pod serving the `tkn` CLI will now be scheduled on nodes, based on the node selector and toleration limits specified in the `TektonConfig` custom resource.
-// https://issues.redhat.com/browse/SRVKP-1804
\ No newline at end of file
+// https://issues.redhat.com/browse/SRVKP-1804
+
+
+[id="release-notes-1-6-2_{context}"]
+== Release notes for {pipelines-title} General Availability 1.6.2
+
+[id="fixed-issues-1-6-2_{context}"]
+=== Fixed issues
+
+* The Operator ensures that only one instance of each type of `TektonInstallerSet` exists after an upgrade.
+// https://issues.redhat.com/browse/SRVKP-1926
+
+* To decide resource recreation during an upgrade, the Operator uses the operator version instead of component version.
```

Comment:
```suggestion
* Before this update, all the reconcilers in the Operator used the component version to decide resource recreation during an upgrade to Red Hat OpenShift Pipelines 1.6.1 from an older version. As a result, resources were not recreated whose component versions did not change in the upgrade. With this update, the Operator uses the Operator version instead of the component version to decide resource recreation during an upgrade.
```
My bit. Feel free to improvise on this.